src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -646,7 +698,7 @@ protected override void SaveCore(ModelSaveContext ctx)
         }
 
         // REVIEW: Destroy.
-        private static int NonZeroCount(ref VBuffer<Float> vector)
+        private static int NonZeroCount(ref VBuffer<float> vector)
         {
             if (!vector.IsDense)
                 return vector.Count;
```

Comment:
Not related to your changes, but I think this is wrong. 
Also, great review comment for this method!

Reply:
I do think it's a pretty extreme review comment! haha


---
In reply to: [220027114](https://github.com/dotnet/machinelearning/pull/957#discussion_r220027114) [](ancestors = 220027114)

Reply:
Should we file an issue on this?

---
In reply to: [220044369](https://github.com/dotnet/machinelearning/pull/957#discussion_r220044369) [](ancestors = 220044369,220027114)